packages/system/src/Stream/Stream/done.ts

```diff
@@ -1,9 +1,9 @@
 import { done as _ } from "../../Effect"
 import type { Exit } from "../../Exit"
-import type { SyncE } from "./definitions"
+import type { IO } from "./definitions"
 import { fromEffect } from "./fromEffect"
 
 /**
- * The stream that ends with the {@link Exit} value `exit`.
+ * The stream that ends with the `Exit` value `exit`.
```

Comment:
why not link to Exit in jsdoc?

Reply:
links are not properly shown in vscode highlight no idea why....

![Screenshot 2020-09-30 114449](https://user-images.githubusercontent.com/24249610/94675891-60027300-0312-11eb-8845-62b494b6bf00.png)

Reply:
oh, i see. but I guess the benefit is that they allow jumping to the Exit interface with ctrl-click

Reply:
Jumping how?

Reply:
IntelliJ allows jumping to definition, when you ctrl-click on the interface/class wrapped into `{@link }` tag